LibreHardwareMonitorLib/Hardware/RyzenSMU.cs

```diff
@@ -0,0 +1,625 @@
+        {
+            {0x001E0004, new Dictionary<uint, SmuSensorType>()
+                {
+                    // {61, new SmuSensorType() { Name = "CPU Core", Type = SensorType.Voltage } },
+                    {62, new SmuSensorType() { Name = "CPU Core", Type = SensorType.Current } },
+                    {63, new SmuSensorType() { Name = "CPU Core", Type = SensorType.Power } },
+                    // {65, new SmuSensorType() { Name = "CPU SoC", Type = SensorType.Voltage } },
+                    {66, new SmuSensorType() { Name = "CPU SoC", Type = SensorType.Current } },
+                    {67, new SmuSensorType() { Name = "CPU SoC", Type = SensorType.Power } },
+                    // {96, new SmuSensorType() { Name = "Core #1", Type = SensorType.Power } },
+                    // {97, new SmuSensorType() { Name = "Core #2", Type = SensorType.Power } },
+                    // {98, new SmuSensorType() { Name = "Core #3", Type = SensorType.Power } },
+                    // {99, new SmuSensorType() { Name = "Core #4", Type = SensorType.Power } },
+                    {108, new SmuSensorType() { Name = "Core #1", Type = SensorType.Temperature } },
+                    {109, new SmuSensorType() { Name = "Core #2", Type = SensorType.Temperature } },
+                    {110, new SmuSensorType() { Name = "Core #3", Type = SensorType.Temperature } },
+                    {111, new SmuSensorType() { Name = "Core #4", Type = SensorType.Temperature } },
+                    {150, new SmuSensorType() { Name = "CPU GFX", Type = SensorType.Voltage} },
+                    {151, new SmuSensorType() { Name = "CPU GFX", Type = SensorType.Temperature} },
+                    {154, new SmuSensorType() { Name = "CPU GFX", Type = SensorType.Clock } },
+                    {156, new SmuSensorType() { Name = "CPU GFX", Type = SensorType.Load } },
+                    {166, new SmuSensorType() { Name = "FCLK", Type = SensorType.Clock } },
+                    {177, new SmuSensorType() { Name = "UCLK", Type = SensorType.Clock } },
+                    {178, new SmuSensorType() { Name = "MCLK", Type = SensorType.Clock } },
+                    {342, new SmuSensorType() { Name = "Display Count", Type = SensorType.Factor } },
+                }
+            }
+        };
+
+        private enum Status : uint
+        {
+            OK = 0x01,
+            Failed = 0xFF,
+            UnknownCmd = 0xFE,
+            CmdRejectedPrereq = 0xFD,
+            CmdRejectedBusy = 0xFC
+        }
+
+        private enum CpuCodeName
+        {
+            UNDEFINED,
+            COLFAX,
+            RENOIR,
+            PICASSO,
+            MATISSE,
+            THREADRIPPER,
+            CASTLEPEAK,
+            RAVENRIDGE,
+            RAVENRIDGE2,
+            SUMMITRIDGE,
+            PINNACLERIDGE,
+            REMBRANT,
```

Comment:
Rambran**d**t 😊 Can we also name these in normal casing? I don't see the need here for caps.

Reply:
Sure, fixed.